WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -363,6 +381,8 @@ private extension SettingsViewController {
         }
     }
 
+    func sitePluginsWasPressed() {}
```

Comment:
nit: Probably better to add a `TODO: To be implemented later #GH-issue` so we communicate with the rest of the team why this method is empty.